### PR TITLE
Update documentation on importing seed database

### PR DESCRIPTION
**Pull request by sandertan**
# What? Why?
The documentation regarding importing the seed database is outdated and the step-by-step tutorial could also be improved. I updated this, which will hopefully make it easier for new users to load a seed database.

### CODE REVIEW

File: docs/Import-the-Seed-Database.md [hunk 1]
```diff
@@ -1,32 +1,43 @@
...
+3. Import the protein database (PDB) part of the seed database. This will enable the visualization of PDB structures in the mutation tab. Loading this file takes more time than loading the previous files, and is optional for users that do not require PDB structures.
+
+    ```
+    mysql --user=cbio_user --password=somepassword cbioportal < seed-cbioportal_hg19_only-pdb.sql
+    ```
+    *Important:* Replace `seed-cbioportal_hg19_only-pdb.sql` with the downloaded version of the PDB database, such as `seed-cbioportal_hg19_v2.1.0_only-pdb.sql`.
+
+:information_source: Please be aware of the version of the seed database. In the [README on datahub](https://github.com/cbioportal/datahub/blob/master/seedDB/README.md), we stated which version of cBioPortal is compatible with the current seed database.
+
+If the database is older from what cBioPortal is expecting, the system will ask you (during startup or data loading) to migrate the database to a newer version. The migration process is described [here](Updating-your-cBioPortal-installation.md#running-the-migration-script).
```
Comment (pieterlukasse) [inline docs/Import-the-Seed-Database.md]: rephrase to "If the database is older than what cBioPortal is expecting..."

File: docs/Import-the-Seed-Database.md [hunk 2]
```diff
@@ -1,32 +1,43 @@
...
+    ```
 
-and:
+2. Import the main part of the seed database:
 
-    > mysql --user=cbio_user --password=somepassword cbioportal  < seed-cbioportal_no-pdb_hg19.sql
-    
-and (this command takes a bit longer to import PDB data that will enable the visualization of PDB structures in the mutation tab): 
+    ```
+    mysql --user=cbio_user --password=somepassword cbioportal < seed-cbioportal_hg19.sql
```
Comment (pieterlukasse) [inline docs/Import-the-Seed-Database.md]: I would change this to 
```
mysql --user=cbio_user --password=somepassword cbioportal < seed-cbioportal_hg19_vX.Y.Z.sql
```
to make the version part more clear

Comment (sandertan) [inline docs/Import-the-Seed-Database.md]: Good suggestion.

File: docs/Import-the-Seed-Database.md (hunk 1 above)
Comment (sandertan) [inline docs/Import-the-Seed-Database.md]: Done

**Comment by sandertan** [comment]
@pieterlukasse I added your suggested improvements, can you merge?

File: docs/Import-the-Seed-Database.md [hunk 3]
```diff
@@ -1,32 +1,43 @@
...
+    ```
 
-    > mysql --user=cbio_user --password=somepassword cbioportal  < seed-cbioportal_only-pdb.sql
+    *Important:* Replace `seed-cbioportal_hg19.sql` with the downloaded version of the seed database, such as `seed-cbioportal_hg19_v2.1.0.sql`.
 
-:information_source: please be aware of the version of the seed DB. If it is different from what the cBioPortal system is expecting, the system will at some point ask you to run a migration step. The system will automatically give you a clear message (with instructions) about this (during startup or during data loading process) if a migration is needed. 
+3. Import the protein database (PDB) part of the seed database. This will enable the visualization of PDB structures in the mutation tab. Loading this file takes more time than loading the previous files, and is optional for users that do not require PDB structures.
+
+    ```
+    mysql --user=cbio_user --password=somepassword cbioportal < seed-cbioportal_hg19_only-pdb.sql
```
Comment (pieterlukasse) [inline docs/Import-the-Seed-Database.md]: x.y.z?

File: docs/Import-the-Seed-Database.md [hunk 4]
```diff
@@ -1,32 +1,43 @@
...
+    ```
+    *Important:* Replace `seed-cbioportal_hg19_only-pdb.sql` with the downloaded version of the PDB database, such as `seed-cbioportal_hg19_v2.1.0_only-pdb.sql`.
```
Comment (pieterlukasse) [inline docs/Import-the-Seed-Database.md]: same here.

File: docs/Import-the-Seed-Database.md (hunk 3 above)
Comment (sandertan) [inline docs/Import-the-Seed-Database.md]: Done

File: docs/Import-the-Seed-Database.md (hunk 4 above)
Comment (sandertan) [inline docs/Import-the-Seed-Database.md]: Done

**Comment by sandertan** [comment]
@pieterlukasse I fixed the issues.

**Comment by pieterlukasse** [comment]
@sandertan can you update the branch?

**Pull request merged**